xla/service/gpu/gemm_rewriter_triton_test.cc

```diff
@@ -82,7 +83,8 @@ ENTRY e {
     lhs_batch_dims={0}, rhs_batch_dims={0}
 })")
                     .value();
-  EXPECT_TRUE(GemmRewriterTriton({se::CudaComputeCapability::AMPERE, 0})
+  GpuVersion gpu_version;
```

Comment:
It looks to me that this is a non-functional change. At least I don't see that GpuVersion is automatically initialized to AMPERE, 0.

Reply:
Much appreciate for the review! This change is for gemm_rewriter_triton's GpuVersion, which makes it more genereal and easier to extend in the future.

I have changed the test part to initialize ```{se::CudaComputeCapability::AMPERE, 0}```.